### DISCUSSION TITLE
HELP! How to solve two .. coexisting in bindings?

### DISCUSSION THREAD

**Discussion by Jim2757** [Budibase/budibase#12341] [Help] [answered]
dear all, if I want to bind the ".id" in the bindings, is there any way to achieve this ?
![2112d6fd-ecf5-4440-820f-ebc16d887ea3](https://github.com/Budibase/budibase/assets/112788102/ac07eba6-3fc3-4fd6-99c5-3df6911d178e)
the raw data is include the item of ".id"  and I urgently need its value
I try to use the "" '' `` to enclose it but they're not work
is it a bug?

**Comment by melohagan** [comment]
Hey @Jim2757 

To get the **Budibase row _id**, the binding would be: `{{ Clicked row._id }}`

If you want to reference another field in your table called **id**, then make sure you don't have the double dot that you currently do.

**Reply by Jim2757** [reply, marked as answer]
solved it, use [] enclose the item; thanks!

**Comment by Jim2757** [comment]
{{bindings.[.item] }}
